src/flag_gems/patches/patch_vllm_all.py

```diff
@@ -338,6 +338,32 @@ def custom_get_scheduler_metadata(
     )
 
 
+def custom_per_token_group_fp8_quant(
+    input: torch.Tensor,
+    output_q: torch.Tensor,
+    output_s: torch.Tensor,
+    group_size: int,
+    eps: float,
+    fp8_min: float,
+    fp8_max: float,
+    scale_ue8m0: bool = False,
+):
+    from flag_gems.ops.per_token_group_quant_fp8 import per_token_group_quant_fp8
```

Comment:
This should work, right?

```suggestion
    from flag_gems.ops import per_token_group_quant_fp8
```